src/lms-ds-loader/.pylintrc

```diff
@@ -0,0 +1,14 @@
+[MASTER]
+ignore = .venv
+
+[MESSAGES CONTROL]
+disable =
+    C0330,
+    C0326,
+    fixme,
+    line-too-long,
+    missing-function-docstring,
+    missing-module-docstring
```

Comment:
We could create tickets for adding docstrings and remove the related "disable" lines in all projects.

Reply:
Actually, I was intending to add docstrings everywhere before completing the ticket. For me that's part of the definition of done. I just copied this file from somewhere else and didn't inspect it closely enough. I'm going to remove lines 10 and 11..

Reply:
Removed those. Will add docstrings before the PR is merged..

Reply:
Docstrings have been added.